src/test/datascience/.vscode/settings.json

```diff
@@ -30,11 +30,12 @@
     "python.experiments.enabled": true,
     // See https://github.com/microsoft/vscode-jupyter/issues/10258
     "jupyter.forceIPyKernelDebugger": false,
-    "python.defaultInterpreterPath": "",
+    "python.defaultInterpreterPath": "/Users/donjayamanne/Development/vsc/vscode-jupyter/.venv/bin/python",
```

Comment:
looks like this path was left here my mistake?

Reply:
Fixed